app/controllers/finding_aids_controller.rb

```diff
@@ -51,5 +51,8 @@ def get_filters(aids)
   private
     def set_finding_aid
       @finding_aid = FindingAid.find(params[:id])
+      @title = @finding_aid.label
+      blockson = Collection.find_by_slug("blockson_collection")
+      @aeon = @finding_aid.collections.include?(blockson) ? true : false
```

Comment:
You don't need the `? true : false` here, the `include?` method will give you the same result.